### PR TITLE
Rename NavigationTracker -> AppTrackingTriggers to better reflect usage

### PR DESCRIPTION
**Pull request by alebedev**
## Describe your changes

Subj. Less confusing name, since the component is responsible for multiple events, not just navigation

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Comment by alebedev** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4293?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#4293** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4293?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#4292** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4292?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @alebedev and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: apps/store/src/services/Tracking/Tracking.ts [hunk 1]
```diff
@@ -69,6 +69,7 @@ export class Tracking {
   private logger = datadogLogs.getLogger(Tracking.LOGGER_NAME)!
 
   public reportAppInit = (countryCode: CountryCode) => {
+    console.debug(`app init, country=${countryCode}`)
```
Comment (Youakeem) [inline apps/store/src/services/Tracking/Tracking.ts:72]: Leftover?

**Review by Youakeem — APPROVED** [review]

File: apps/store/src/services/Tracking/Tracking.ts (hunk 1 above)
Comment (alebedev) [reply apps/store/src/services/Tracking/Tracking.ts:72]: It's intentional - we log other events we track

**Comment by alebedev** [comment]
### Merge activity

* **May 20, 8:05 AM EDT**: @alebedev started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4293).

File: apps/store/src/services/Tracking/Tracking.ts (hunk 1 above)
Comment (Youakeem) [reply apps/store/src/services/Tracking/Tracking.ts:72]: Got it!

**Pull request merged**